.gitignore

```diff
@@ -34,6 +34,7 @@ nosetests.xml
 coverage.xml
 
 # Translations
+*.pot
 *.mo
```

Comment:
I think we will need to remove .mo files from the .gitignore if we want to commit translations. WDYT @jmaupetit?

Reply:
Will we not commit the `.po` files? I think `.mo` is a non-human-readable format that can be produced from `.po` and therefore not useful to commit?

Reply:
yes. But then we need to add compilemessages to the build in circleci ;-)

Reply:
Yup. `.mo` files should be in the repository and bundled in the distributed image/package.